org-cyf-piscine/content/sprints/1/day-plan/index.md

```diff
@@ -0,0 +1,49 @@
++++
+title = 'day-plan'
+layout = 'day-plan'
+emoji= '🧑🏾‍🤝‍🧑🏾'
+menu_level = ['sprint']
+weight = 3
+[[blocks]]
+name="Induction"
+src="module/piscine/induction"
+time=15
+[[blocks]]
+name="Energiser"
+src="energisers/zip-zap-boing"
+[[blocks]]
+name="Morning break"
+src="blocks/morning-break"
+[[blocks]]
+name="Kickoff"
+src="module/piscine/kickoff"
+time=20
+[[blocks]]
+name="First group project"
+src="https://github.com/CodeYourFuture/The-Piscine/tree/main/Project-1"
+time=45
+[[blocks]]
+name="Development"
+src="module/piscine/development"
+time="60"
+[[blocks]]
+name="lunch"
+src="blocks/lunch"
+[[blocks]]
+name="Development"
+src="module/piscine/development"
+time="75"
+[[blocks]]
+name="Afternoon break"
+src="blocks/afternoon-break"
+[[blocks]]
+name="Development"
+src="module/piscine/development"
+time="30"
+[[blocks]]
+name="Demo"
+src="module/piscine/demo"
```

Comment:
One of our open questions was "how do we make sure to stagger demos so that a small number of people can assess a large number of demos".

Maybe we want to somehow signal in the day plan that some people will demo at different times?

Or maybe we actually want to just have two demo sessions each day and have everyone demo in both of them? i.e. in Sprint 1 we'd have one after lunch or between the after-lunch dev block and the afternoon break, _and_ one at the end of the day. In Sprints 2 and 3 we could have a start-of-day demo and an end-of-day demo?

Reply:
Yes, I thought about this too. 

And the answer is I don't know. When we have lots of bitty sessions, in reality they don't really happen. People spend a lot of time in breaks and lunch and faffing about in the morning, so really they just get 2 sessions of anything really done. 

Mull on it!